fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -244,6 +244,20 @@ class RestoreConfig : public KeyBackedConfig {
 
 	Key applyMutationsMapPrefix() { return uidPrefixKey(applyMutationsKeyVersionMapRange.begin, uid); }
 
+	ACTOR static Future<Version> getCurrentVersion_impl(Reference<ReadYourWritesTransaction> tr, UID uid) {
+		state Future<Optional<Value>> beginVal = tr->get(uidPrefixKey(applyMutationsBeginRange.begin, uid), true);
```

Comment:
When you are only creating / waiting on one future at a time you can do it directly as follows which simplifies the code a bit:

```
Optional<Value>> beginVal = wait(tr->get(uidPrefixKey(applyMutationsBeginRange.begin, uid), true));
```
 so then you can use `beginVal.present()` and `.get()` more directly.

Reply:
Thanks, done.